eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -46,6 +47,28 @@ case class RemoteCommit(index: Long, spec: CommitmentSpec, txid: ByteVector32, r
 case class WaitingForRevocation(nextRemoteCommit: RemoteCommit, sent: CommitSig, sentAfterLocalCommitIndex: Long, reSignAsap: Boolean = false)
 // @formatter:on
 
+trait ChannelCommitments {
```

Comment:
What is the rationale for the naming? The following may sound a bit java-ish, but at least it conveys the information that `Commitments <: AbstractCommitments`.

```suggestion
trait AbstractCommitments {
```

Reply:
I tried to convey it's something, well, abstract so this also sounds good.